tests/atomistics/structure/test_atoms.py

```diff
@@ -854,6 +854,14 @@ def test_get_bonds(self):
         # print H2O.get_bonds(radius=2.)[0]
         # print np.sum(H2O.get_masses())/H2O.get_volume()
 
+    def test_get_symmetr(self):
+        cell = 2.2 * np.identity(3)
+        Al = Atoms("AlAl", scaled_positions=[(0, 0, 0), (0.5, 0.5, 0.5)], cell=cell)
+        with self.assertRaises(ValueError):
+            Al.symmetrize_vector(1)
+        v = np.random.rand(6).reshape(-1, 3)
+        self.assertAlmostEqual(np.linalg.norm(Al.symmetrize_vector(v)), 0)
```

Comment:
Maybe testing this for a structure with absolutely no symmetry would also be useful (if it's not too hard)

Reply:
That would make the tests slower because it would require a large box to destroy all symmetries

Reply:
I included a case where I destroy the box symmetry in the direction of x.